db/warehouse/migrate/20231103165752_add_hmis_activity_log.rb

```diff
@@ -0,0 +1,21 @@
+class AddHmisActivityLog < ActiveRecord::Migration[6.1]
+  def change
+    # Should this go in the app db?
+    # Index on user id and ds id?
+
+    create_table :hmis_activity_logs do |t|
+      t.references :user, null: false, index: true
+      t.references :data_source, null: false
+      t.string :ip_address, null: false
+      t.string :session_hash
+      t.string :referer # raw referer
+      t.string :path # cleaned/decrypted path
+      t.string :operation_name # GraphQL operation name
+      t.string :variables # GraphQL variables (JSON)
```

Comment:
Totally not necessary, but maybe useful:
```suggestion
      t.string :referer, comment: 'raw referer'
      t.string :path, comment: 'cleaned/decrypted path'
      t.string :operation_name, comment: 'GraphQL operation name'
      t.string :variables, comment: 'GraphQL variables (JSON)'
```